wix/Build.OpenJDK_generic.cmd

```diff
@@ -178,7 +178,7 @@ FOR %%A IN (%ARCH%) DO (
 
 	REM Build without extra Source Code feature
 
-    IF !PLATFORM! == x64 (
+    IF !PLATFORM! == x64 IF !PRODUCT_MAJOR_VERSION! == 8 (
```

Comment:
I would nest that 2nd IF to match the style of the rest of this file

Reply:
Done